ports/json-schema-validator/portfile.cmake

```
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO pboettch/json-schema-validator
+    REF 27fc1d094503623dfe39365ba82581507524545c
+    SHA512 4fd05087743f43871586a53d119acd1a19d0bdec8a5620f62b6eee7a926d285842e8439127eec52eeb11069c92b8d9af28558897d48e2422ecafca39d9f23cdb
+    HEAD_REF master
+    PATCHES
+        "${CMAKE_CURRENT_LIST_DIR}/fix-ambiguous-assignment.patch"
```

```suggestion
       fix-ambiguous-assignment.patch
```
Could you please update this?

---

Ack.